### PR TITLE
Add io.github.kriptolix.Poliedros

### PR DESCRIPTION
**Pull request by kriptolix**
### Please confirm your submission meets all the criteria

- [x] Please describe the application briefly. 
Poliedros is a multi-type dice roller.
- [x] The domain used for the application ID is https://github.com/kriptolix/Poliedros and the [application id guidelines][appid] are followed.
- [x] I have read and followed all the [Submission requirements][reqs] and the [Submission guide][reqs2].
- [x] I have [built][build] and tested the submission locally.
- [x] I am an author/developer/upstream contributor to the project. If not, I contacted upstream developers about this submission. **Link:**

[appid-domain]: https://docs.flathub.org/docs/for-app-authors/requirements/#control-over-domain-or-repository
[appid]: https://docs.flathub.org/docs/for-app-authors/requirements#application-id
[reqs]: https://docs.flathub.org/docs/for-app-authors/requirements
[reqs2]: https://docs.flathub.org/docs/for-app-authors/submission
[build]: https://docs.flathub.org/docs/for-app-authors/submission/#before-submission

### CODE REVIEW

File: io.github.kriptolix.Poliedros.json [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    "modules" : [
+        {
+            "name" : "poliedros",
+            "builddir" : true,
+            "buildsystem" : "meson",
+            "sources" : [
+                {
+                    "type" : "git",
+                    "url" : "https://github.com/kriptolix/Poliedros.git",
+                    "tag" : "v1.0.0"
```
Comment (hfiguiere) [inline io.github.kriptolix.Poliedros.json:33]: the tag doesn't seem to exist.

Comment (kriptolix) [reply io.github.kriptolix.Poliedros.json:33]: Sorry, I created the tag.

**Comment by hfiguiere** [comment]
bot, build io.github.kriptolix.Poliedros

**Pull request closed without merging**